docs/docs/command-line-interface.mdx

```diff
@@ -574,6 +574,7 @@ identical training examples.
 The validator also checks if you have any stories where different assistant actions follow from the same
 dialogue history. Conflicts between stories will prevent a model from learning the correct
 pattern for a dialogue.
+To learn more about the checks performed by the validator on the business logic, continue reading in the [next section](./command-line-interface.mdx#validate-flows).
```

Comment:
I think we should decide on whether we say business logic or flows to make it more consistent (line 565 says flows). I'd opt for flows